package.json

```diff
@@ -15,6 +15,8 @@
     "react-router-dom": "^6.4.3",
     "react-scripts": "5.0.1",
     "uuid": "^9.0.0",
+    "redux-thunk": "^2.4.2",
```

Comment:
- [ ] Great work on your project! I did notice, however, that there seems to be an issue with the link to the [MIT license](https://github.com/ivanmvh/bookstore-ivanmvh/blob/BR5-BOOKSTORE-CONNECT-REDUX-TO-API/MIT.md) in your README. Upon clicking on the link, it appears to redirect to a page that is not found. I kindly request that you take a moment to update the link, or if it has not yet been added, include it and personalize it to match the specifics of your project. Thank you for your attention to this matter.📝🔍🛠

Reply:
I also fix other things in the readme.md file.Thanks